repositories/PageRepository.js

```diff
@@ -86,5 +86,13 @@ module.exports.undelete = function(id) {
 };
 
 module.exports.move = ({ id, sectionId, position }) => {
-  return db.transaction(trx => movePage(trx, { id, sectionId, position }));
+  return db.transaction(trx =>
+    movePage(trx, { id, sectionId, position }).then(fromDb)
+  );
+};
+
+module.exports.getPosition = ({ id }) => {
+  return Page.findById(id).then(({ SectionId }) =>
```

Comment:
what's the reason for capitalising `SectionId` here?

Reply:
Because I made the foolish design choice right at the start of this project to use PascalCase for foreign keys in the DB, but expose them as camelCase to clients.

I could add a `then(fromDb)` here to avoid it. But felt a bit much since it's only a temporary value, and this method returns a single integer (not a row or set of rows)